op-batcher/batcher/driver.go

```diff
@@ -148,8 +148,13 @@ func (l *BatchSubmitter) Stop() {
 // 4. Load all new blocks into the local state.
 func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context) {
 	start, end, err := l.calculateL2BlockRangeToStore(ctx)
-	if err != nil {
-		l.log.Trace("was not able to calculate L2 block range", "err", err)
+	if errors.Is(err, ErrReorg) {
+		l.log.Warn("Found L2 reorg", "err", err)
```

Comment:
nit: now also include `err` variable in `log.Warn` in loop below, as is done here